packages/fuelet_secure_layer/lib/src/features/account/repository/accounts_local_repository_impl.dart

```diff
@@ -69,7 +70,11 @@ class AccountsLocalRepositoryImpl implements IAccountsLocalRepository {
       futures.add(accountsBox.put(account.fuelAddress.bech32Address, account));
     }
     await Future.wait(futures);
-    await _privateDataRepository.flushData();
+    final accountAddressesToFlush = accounts
```

Comment:
there's already a loop on line 68 so instead of iterating twice over the same `List` you might consider building a set there like:
```dart
final accountAddressesToFlush = <String>{};
for (Account account in accounts) {
      account = _replaceForbiddenSymbolsIfNeeded(account);
      if (account.isOwner) {
          accountAddressesToFlush.add(account.fuelAddress.bech32Address);
      }
      futures.add(accountsBox.put(account.fuelAddress.bech32Address, account));
}
```

Reply:
agree, fixed